components/types/gameSchemas.ts

```diff
@@ -104,3 +104,51 @@ export type LoadSaveBody = Partial<{
     difficultyLevel: number
     contractId: string
 }>
+
+/**
+ * Query params that `/profiles/page/Safehouse` gets.
+ * Roughly the same as {@link SafehouseCategoryQuery} but this route is only for H1.
+ */
+export type SafehouseQuery = {
```

Comment:
I love these, we should do this for all endpoints. A separate query type (and a typed response preferably).